### PR TITLE
fix(CX-2342): fixed appearance o MyCollection placeholder

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [CX-2342]

### Description

Fixed appearance o MyCollection placeholder.
It was pushed a bit up

"After" column show hot it looks in [the first commit](https://github.com/artsy/eigen/pull/6599/commits/f31229f0906191d715170cbefb2c5dc96266a6c9) (a bit jumpy)
In the final version I have decided not to include these changes
| in the first commit | final (with personal data) | final (without personal data)|
| -| - | - |
| https://user-images.githubusercontent.com/36167539/162793963-875085a2-5572-43bc-b9ca-fd96b00b8fd6.mp4 | https://user-images.githubusercontent.com/36167539/162794663-3e507015-b6b2-48ce-9e47-3f5e76472e0b.mp4 | https://user-images.githubusercontent.com/36167539/162795029-5074235c-437f-4174-a3e2-72d43d889510.mp4 |

| hi | hello | hey |
### PR Checklist (tick all before merging)

- [x] Tested on **iOS** and **Android**
- [x] Screenshots and videos 
- [ ] Added Tests and Stories
- [ ] App state migration
- [ ] Feature flag

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-  fixed appearance o MyCollection placeholder -daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-2342]: https://artsyproduct.atlassian.net/browse/CX-2342?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/MyCollection/MyCollection.tsx [hunk 1]
```diff
@@ -320,25 +320,22 @@ export const MyCollectionPlaceholder: React.FC<{}> = () => {
...
             <PlaceholderBox width={100} height={100} borderRadius={50} />
             <Flex justifyContent="center" ml={2}>
-              <PlaceholderText width={80} height={24} />
+              <PlaceholderText width={80} height={35} />
+              <PlaceholderText width={100} height={35} />
               <PlaceholderText width={100} />
             </Flex>
           </Flex>
-          <Spacer mb={1} />
-          <Spacer mb={1} />
```
Comment (rajsam003) [inline src/app/Scenes/MyCollection/MyCollection.tsx:334]: Why was it used twice instead of once `<Spacer mb={2} />` 🤔

**Review by rajsam003 — APPROVED** [review]
LGTM 👏

File: src/app/Scenes/MyCollection/MyCollection.tsx (hunk 1 above)
Comment (dariakoko) [reply src/app/Scenes/MyCollection/MyCollection.tsx:334]: Don't know 🤷‍♀️
I've changed it in my pr to

**Review by MounirDhahri — APPROVED** [review]
Thank you Daria!

**Comment by lordkiz** [comment]
This looks good @dariakoko . 👏🏾
This fixes the loading skeleton issue. How about the refresh loading indicator? We can always address that in another PR.

**Comment by dariakoko** [comment]
> This looks good @dariakoko . 👏🏾 This fixes the loading skeleton issue. How about the refresh loading indicator? We can always address that in another PR.

Thanks @lordkiz! I think the loading indicator was fixed in this PR, wasn't it? https://github.com/artsy/eigen/pull/6519

**Comment by lordkiz** [comment]
> Thanks @lordkiz! I think the loading indicator was fixed in this PR, wasn't it? #6519

This looks like that PR centered the LoadFailure view. I meant when you pull the mycollection screen down, while the artwork is reloading, there is no spinner above to show it is reloading.

**Comment by dariakoko** [comment]
> > Thanks @lordkiz! I think the loading indicator was fixed in this PR, wasn't it? #6519
> 
> This looks like that PR centered the LoadFailure view. I meant when you pull the mycollection screen down, while the artwork is reloading, there is no spinner above to show it is reloading.

Oh, okay, got it. I will address this in another PR.

**Pull request merged**